### ISSUE TITLE
Searching for "node" yields an internal server error

### ISSUE THREAD

**Issue by spanierm** [styfle/packagephobia#142]
Nice work but I stumbled upon an error:

Searching for [node](https://packagephobia.now.sh/result?p=node) yields an internal server error:
![2018-09-29-145848_617x516_scrot](https://user-images.githubusercontent.com/21294634/46246063-4e6ef380-c3f8-11e8-8704-c5f964170f3e.png)

**Comment by styfle** [comment]
Thanks for the bug report!

It looks like this started happening recently because all version of node are failing even though there are many versions that were recorded in the past as you can see here: https://packagephobia.now.sh/result?p=node@10.0.0

The error message is as follows:

```
ERROR: Command failed: npm i --no-package-lock --no-audit node@9.11.2
/bin/sh: npm: not found
internal/modules/cjs/loader.js:583
    throw err;
    ^
Error: Cannot find module 'node-linux-x64/package.json'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:581:15)
    at Function.resolve (internal/modules/cjs/helpers.js:30:19)
    at ChildProcess.<anonymous> (/tmp/tmp-package0.4853848379295722/node_modules/node-bin-setup/index.js:18:27)
    at ChildProcess.emit (events.js:182:13)
    at maybeClose (internal/child_process.js:961:16)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:250:5)
npm WARN tmp-package0.4853848379295722@1.0.0 No description
npm WARN tmp-package0.4853848379295722@1.0.0 No repository field.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! node@9.11.2 preinstall: `node installArchSpecificPackage`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the node@9.11.2 preinstall script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2018-09-29T14_45_12_884Z-debug.log
```

This is kind of strange because [node-linux-x64](https://packagephobia.now.sh/result?p=node-linux-x64) seems to work just fine.

**Comment by styfle** [comment]
This might have broke around the time that Dockerfile was introduced.
Prior to that I was using whatever system zeit had for node/npm deployments.
In early June (PR #85), I introduced docker deployments. That seems to correlate to the last known version of node that was installed and sized properly: `node@10.2.0` was released late May.

So maybe the answer is to change the Dockerfile to install some missing build tools?
